### DISCUSSION TITLE
安卓用戶注意！5款APP暗藏木馬病毒 200萬人恐受害

### DISCUSSION THREAD

**Discussion by YuHengChen54** [1101-GS3073-C/Python-Community-GS3073-C#445] [General]
常常看到甚麼什麼程式有病毒，但感覺這些程式都是很少人會安裝的哈哈哈哈
https://tw.news.yahoo.com/%E5%AE%89%E5%8D%93%E7%94%A8%E6%88%B6%E6%B3%A8%E6%84%8F-5%E6%AC%BEapp%E6%9A%97%E8%97%8F%E6%9C%A8%E9%A6%AC%E7%97%85%E6%AF%92-200%E8%90%AC%E4%BA%BA%E6%81%90%E5%8F%97%E5%AE%B3-003910133.html

**Comment by lx3211** [comment]
我很好奇play商店在上架之前沒有檢查的機制嗎

**Comment by Chenszuhong** [comment]
感覺太扯了吧，好險我是用蘋果😂😂

**Comment by BingRong1217** [comment]
隨著資訊科技的發展，資安問題也應該被重視

**Comment by Ben6uo** [comment]
雖然安卓是比IOS方便但那是由資安那裡扣出來的

**Comment by limitx0** [comment]
安卓手機真的是比較不安全
